2021/ashok2022re/journal/metadata.yaml

```diff
@@ -0,0 +1,162 @@
+# To be filled by the author(s) at the time of submission
+# -------------------------------------------------------
+
+# Title of the article:
+#  - For a successful replication, it should be prefixed with "[Re]"
+#  - For a failed replication, it should be prefixed with "[¬Re]"
+#  - For other article types, no instruction (but please, not too long)
+# Change the default title
+title: "[Re] Does Self-Supervision Always Improve Few-Shot Learning?"
+
+# List of authors with name, orcid number, email and affiliation
+# Affiliation "*" means contact author (required even for single-authored papers)
+authors:
+  - name: Arjun Ashok
+    orcid: 0000-0002-2752-2509 
+    email: arjun.ashok.psg@gmail.com
+    affiliations: 1,*
+
+  - name: Haswanth Aekula
+    orcid: 0000-0001-8058-3602
+    email: haswanth.kumar.39@gmail.com
+    affiliations: 2      # * is for contact author
+
+# List of affiliations with code (corresponding to author affiliations), name
+# and address. You can also use these affiliations to add text such as "Equal
+# contributions" as name (with no address).
+affiliations:
+  - code:    1
+    name:    PSG College of Technology, India
+    # address: Avinashi Rd, Peelamedu, Coimbatore, Tamil Nadu 641004
+  - code:    2
+    name:    DTICI, India
+    # address: Plot No. 9 & 10, 1, Whitefield Main Rd, near Satya Sai Baba Hospital, EPIP Zone, Whitefield, Bengaluru, Karnataka 560066
+
+
+# List of keywords (adding the programming language might be a good idea)
+keywords:  rescience c, machine learning, deep learning, python, pytorch, few-shot learning, self-supervised learning 
+
+
+# Code URL and DOI/SWH (url is mandatory for replication, doi after acceptance)
+# You can get a DOI for your code from Zenodo, or an SWH identifier from
+# Software Heritage.
+#   see https://guides.github.com/activities/citable-code/
+code:
+  - url: https://github.com/ashok-arjun/MLRC-2021-Few-Shot-Learning-And-Self-Supervision/
+  - doi: 10.5281/zenodo.6508499
+  - swh: swh:1:dir:aaaaaaaaaaaaaaaaaaaaaaaaaaaaa
```

Comment:
Hi, please provide the correct SWH id ASAP.

Reply:
Done!